_targets/build.project.ia32-generic

```diff
@@ -48,9 +48,11 @@ b_build_target() {
 	ln -f "$PREFIX_ROOTFS/bin/"{psh,reboot}
 	ln -f "$PREFIX_ROOTFS/bin/"{psh,top}
 
+	: ${SYSPAGE='pc-tty pc-ata psh'}
```

Comment:
Nitpicking, to share the good practices (i know the script has other issues to be solved) - use `shellcheck` to avoid potential problems in shell scripts. Here we need to double-quote it because of [SC2223](https://github.com/koalaman/shellcheck/wiki/SC2223).

Personally I would put this value somewhere at the top of the file, so the user would be able to see what can be set using ENV variables at a glance.